openedx/core/djangoapps/api_admin/forms.py

```diff
@@ -0,0 +1,42 @@
+"""Forms for API management."""
+from django import forms
+from django.conf import settings
+from django.utils.translation import ugettext as _
+
+from openedx.core.djangoapps.api_admin.models import ApiAccessRequest
+
+
+class ApiAccessRequestForm(forms.ModelForm):
+    """Form to request API access."""
+
+    company_name = forms.CharField(label=_('The name of your company.'))
+    company_address = forms.CharField(label=_('The contact address of your company.'))
+    terms_of_service = forms.BooleanField(
+        label=_('{platform_name} API Terms of Service').format(platform_name=settings.PLATFORM_NAME),
+        help_text=_(
+            'The resulting Package will still be considered part of Covered Code. Your Grants.'
```

Comment:
Ah, that was slightly unclear -- there's a template, but the markup for the form is generated. You can see the markup here: https://github.com/edx/edx-platform/pull/11982/files#diff-bc7fb254303d3548bf82f7484aaeb87eR14